### PR TITLE
Export functions directly for ES6 solidity helpers

### PR DESCRIPTION
**Pull request by rupurt**
Limit the re-assignment of function declarations on each import

### CODE REVIEW

File: solidity/test/support/helpers.js [hunk 1]
```diff
@@ -82,233 +55,205 @@ let _0x,
...
+  defaultAccount,
+  eth,
+  oracleNode,
+  stranger
+}
 
-  bigNum = function bigNum (number) {
-    return web3.toBigNumber(number)
-  }
+export const _0x = val => `0x${toHex(val)}`
```
Comment (j16r) [inline solidity/test/support/helpers.js]: This was removed.

File: solidity/test/support/helpers.js [hunk 2]
```diff
@@ -82,233 +55,205 @@ let _0x,
...
+export const getEvents = contract => (
+  new Promise(
+    (resolve, reject) =>
+      contract
+        .allEvents()
+        .get((error, events) => (error ? reject(error) : resolve(events)))
+  )
+)
+
+export const lPad = s => {
```
Comment (j16r) [inline solidity/test/support/helpers.js]: I believe this was removed also.

File: solidity/test/support/helpers.js [hunk 3]
```diff
@@ -82,233 +55,205 @@ let _0x,
...
   }
+  return s
+}
 
-  checkPublicABI = function checkPublicABI (contract, expectedPublic) {
-    let actualPublic = []
-    for (const method of contract.abi) {
-      if (method.type === 'function') actualPublic.push(method.name)
-    };
+export const rPad = s => {
```
Comment (j16r) [inline solidity/test/support/helpers.js]: Removed.

**Comment by rupurt** [comment]
@j16r updated with removals

**Pull request merged**